recipe/meta.yaml

```diff
@@ -69,7 +68,6 @@ outputs:
         - cuda-version >=11.2,<12  # [(cuda_compiler_version or "").startswith("11")]
       run_constrained:
         - librmm ={{ rapids_version }}               # [linux and cuda_compiler != "None"]
-        - {{ pin_compatible("nccl", max_pin="x") }}  # [linux and cuda_compiler != "None"]
```

Comment:
Also no need to have `nccl` in `run_constrained` since `nccl`'s `run_exports` are included, which places it in `run` with the proper constraint